staging/src/k8s.io/kubectl/pkg/describe/versioned/describe.go

```diff
@@ -3345,13 +3345,32 @@ func describeNodeResource(nodeNonTerminatedPodsList *corev1.PodList, node *corev
 		corev1.ResourceMemory, memoryReqs.String(), int64(fractionMemoryReqs), memoryLimits.String(), int64(fractionMemoryLimits))
 	w.Write(LEVEL_1, "%s\t%s (%d%%)\t%s (%d%%)\n",
 		corev1.ResourceEphemeralStorage, ephemeralstorageReqs.String(), int64(fractionEphemeralStorageReqs), ephemeralstorageLimits.String(), int64(fractionEphemeralStorageLimits))
+
 	extResources := make([]string, 0, len(allocatable))
+	hugePageResources := make([]string, 0, len(allocatable))
 	for resource := range allocatable {
-		if !resourcehelper.IsStandardContainerResourceName(string(resource)) && resource != corev1.ResourcePods {
+		if resourcehelper.IsHugePageResourceName(resource) {
```

Comment:
Do we want to list Pods here?

Reply:
Guess we can add pids as well.

Now that we have tests it would be much easier to refactor the code and add support for that. :)